Slave.h

```diff
@@ -128,6 +130,11 @@ class Slave
         ext_state.initTableCount(_db_name + "." + _tbl_name);
     }
 
+    void setStructCallback(const std::string& _db_name, const std::string& _tbl_name, struct_callback _callback) {
```

Comment:
In this file braces are usually starting at the next line after function declaration.

Reply:
Ok